README.adoc

```diff
@@ -107,8 +95,7 @@ include::finish/src/main/resources/META-INF/microprofile-config.properties[]
 // =================================================================================================
 == Injecting static configuration
 
-The MicroProfile Config API was added as a dependency to your [hotspot file=0]`pom.xml` file. Look for the dependency with the [hotspot=config_aID file=0]`mpConfig` artifact ID.
-This feature allows you to use the MicroProfile Config API to externalize configurations for your microservices.
+The MicroProfile Config API is included in the MicroProfile dependency specified in your [hotspot file=0]`pom.xml` file. Look for the dependency with the [hotspot=microprofile file=0]`microprofile` artifact ID. This dependency provides a library that allows you to use the MicroProfile Config API to externalize configurations for your microservices.
```

Comment:
Change
"The MicroProfile Config API is included in the MicroProfile dependency specified in your [hotspot file=0]`pom.xml` file."
To
"The MicroProfile Config API is included in the MicroProfile dependency that is specified in your [hotspot file=0]`pom.xml` file."

Reply:
Updated